database_test.go

```diff
@@ -195,8 +195,8 @@ func TestDatabaseClient(t *testing.T) {
 					CreatedBy:      user,
 					LastEditedBy:   user,
 					Parent: notionapi.Parent{
-						Type:   "page_id",
```

Comment:
You don't need to change existing tests, it's still valid.
A better way is to add more test cases.

Reply:
Done